pkg/apis/pipeline/v1alpha1/pipelinerun_types.go

```diff
@@ -91,6 +91,9 @@ type PipelineRunSpec struct {
 	// with those declared in the pipeline.
 	// +optional
 	Workspaces []WorkspaceBinding `json:"workspaces,omitempty"`
+	// PipelineRunTaskSpec holds a set of runtime specs
+	// +optional
+	TaskSpec []PipelineRunTaskSpec `json:"pipelineRunTaskSpec,omitempty"`
```

Comment:
Should `json:"pipelineRunTaskSpec,omitempty"` be ``json:"TaskSpec,omitempty"`?

Did we decide on the name for this field? `TaskSpec` sounds a bit ambiguous. Perhaps `TaskOverrides` or `TaskRuntimeSpec`? @skaegi @vdemeester ?

Reply:
I do like `TaskOverrides` :upside_down_face: but @skaegi @sbwsg @bobcatfish might have other naming proposition :stuck_out_tongue_closed_eyes:

Reply:
I have updated to ```json:"TaskSpec,omitempty"```.  I think both ```TaskOverrides``` and ```TaskRuntimeSpec``` sounds good!

Reply:
It provides custom defaults for the TaskRuns so since a PodSpec provides defaults for Pods.... I'd suggest the name 'taskRunSpec`

Reply:
I have updated with taskRunSpecs now.